src/poliastro/earth/sensors.py

```diff
@@ -1,110 +1,107 @@
-import numpy as np
 from astropy import units as u
 
+from poliastro.core.sensors import (
+    max_and_min_ground_range_with_specific_azimuth as max_and_min_ground_range_with_specific_azimuth_fast,
+    min_and_max_ground_range as min_and_max_ground_range_fast,
+)
+
 
-@u.quantity_input(h=u.km, η_fov=u.rad, η_center=u.rad, R=u.km)
-def min_and_max_ground_range(h, η_fov, η_center, R):
+@u.quantity_input(h=u.km, n_fov=u.rad, n_center=u.rad, R=u.km)
+def min_and_max_ground_range(h, n_fov, n_center, R):
     """
-    Calculates the minimum and maximum values of ground-range angles
+    Calculates the minimum and maximum values of ground-range angles.
+
     Parameters
     ----------
     h: ~astropy.units.Quantity
         Altitude over surface.
-    η_fov: ~astropy.units.Quantity
+    n_fov: ~astropy.units.Quantity
         Angle of the total area that a sensor can observe.
-    η_center: ~astropy.units.Quantity
+    n_center: ~astropy.units.Quantity
         Center boresight angle.
     R: ~astropy.units.Quantity
         Attractor equatorial radius
+
     Returns
     -------
-    Λ_min: ~astropy.units.Quantity
+    lambda_min: ~astropy.units.Quantity
```

Comment:
For some reason, `~astropy.units.Quantity` does not always work:

![Screenshot_2021-05-03 poliastro earth sensors — poliastro 0 15 dev0 documentation(1)](https://user-images.githubusercontent.com/316517/116865062-36435e80-ac09-11eb-8113-434f8950d420.png)